src/components/Pagination/Pagination.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import {
+  Meta,
+  Story,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import * as stories from "./Pagination.stories.tsx";
+
+<Meta
+  title="Pagination"
+  component={Pagination}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17226%3A932",
+    },
+  }}
+/>
+
+# Pagination
+
+<Description of={Pagination} />
+Pagination navigates between the pages of a multi-page application. It is commonly
+in search results. Use controls in the Basic example to explore this component with
+very many or very few pages.
+
+Pagination uses anchor tags because it is navigating between URLs.
+
+<Preview>
+  <Story story={stories.paginationUrlHref} />
```

Comment:
This is linking back to the `canvas` page.  I could make different stories for `canvas` and `docs` to make this link to the `doc` page - would this be helpful or more confusing?

Reply:
Per discussion, this issue will be resolved under a separate documentation-specific ticket.